lib/jekyll/tags/include.rb

```diff
@@ -21,6 +21,10 @@ class IncludeTag < Liquid::Tag
         (?<params>.*)
       !x
 
+      FULL_VALID_SYNTAX = %r!\A\s*(?:#{VALID_SYNTAX}(?=\s|\z)\s*)*\z!
+      VALID_FILENAME_CHARS = %r!^[\w/\.-]+$!
+      INVALID_SEQUENCES = %r![\./][\./]+!
```

Comment:
`%r![./]{2,}!` ?

Reply:
is this faster or just more idiomatic?

Reply:
Easier to read. More obvious what the intent is.